recipe/build_base.bat

```diff
@@ -1,5 +1,9 @@
 @echo on
 
+set "LIB=%LIBRARY_LIB%;%LIB%"
+set "LIBPATH=%LIBRARY_LIB%;%LIBPATH%"
+set "INCLUDE=%LIBRARY_INC%;%INCLUDE%"
```

Comment:
Had wondered about these when they were here before. Now after the configuration issues, am wondering if we pick up a system Python from the VM without them

Let's try adding them back to see if they help

We probably want a comment here as well

Reply:
Doesn't seem to help. Going to drop these again to avoid confusion